src/core/jit/graphql_executor.rs

```diff
@@ -21,15 +23,42 @@ impl JITExecutor {
     }
 }
 
+impl Clone for Lift<async_graphql::Request> {
+    fn clone(&self) -> Self {
+        let mut request = async_graphql::Request::new(self.query.clone());
+        request.variables = self.variables.clone();
+        request.extensions = self.extensions.clone();
+        request.operation_name = self.operation_name.clone();
+        request.into()
+    }
+}
+
 impl Executor for JITExecutor {
     fn execute(&self, request: async_graphql::Request) -> impl Future<Output = Response> + Send {
-        let request = jit::Request::from(request);
+        let request = request.lift();
+        let jit_request = jit::Request::from(request.clone().take());
```

Comment:
Cloning should only be done if it's a introspection query.

Reply:
done